content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,169 @@
+```
+Now we plot these value using the `plot` function.
+```matlab
+subplot(2, 1, 2);
+plot(angle(double(f_sub)));   %angle are the w values.
+title('phase spectrum')   % title
+```
+`plot(angle(double(data_value)))` gives the phase spectrum plot.
+
+![phase spectrum](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-three.png)
+
+Now lets plot the magnitude response using the same values of `w`.
+```matlab
+subplot(2, 1, 1); 
+plot(abs(double(f_sub))); %magnitude spectrum plot
+title('magnitude spectrum')
+```
+The code `plot(abs(double(data_values)))` gives the magnitude spectrum. This plot uses the absolute values of the data, thus `abs`.
+
+![magnitude spectrum](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-four.png)
+
+### Example 2
+lets look at another example;
+![example 2](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-nine.jpeg)
+
+So we will have;
+```Matlab
+syms t w T W
+f = int(1*exp(-j*w*t), t, [-T, T])
+```
+The output here is:
+```Matlab
+f =
+ 
+(2*sin(T*w))/w
+``` 
+pretty(f) gives:
+```Matlab
+2 sin(T w)
+----------
+     w
+```
+Now, we find the values of `w` and then plot these values. To find the values of `w`, we use the `subs` function.
+```Matlab
+f_sub = subs(f, T, 0.5)
+ 
+f_sub =
+ 
+(2*sin(w/2))/w     % the output after execution of the code above.
+```
+We are now plotting the absolute values of the variable `f-sub`.
+```matlab
+ezplot(abs(f_sub), [-pi: pi])
+ezplot(abs(f_sub), [-2*pi: 2*pi])
+ezplot(abs(f_sub), [-4*pi: 4*pi])
+ezplot(abs(f_sub), [-8*pi: 8*pi])
+```
+The output is:
+
+![Range of -pi to pi](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-five.png)
+![Range of -2pi to 2pi](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-six.png)
+![Range of -4pi to 4pi](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-seven.png)
+![Range of -8pi to 8pi](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-eight.png)
+
+While making the plots, we used the `ezplot` function. `ezplot(FUN)` is used to plot function `x` over the default domain, `-2*pi<x<2*pi`.
+
+### Conclusion
+Symbolic toolbox is a very important toolbox in Matlab to solve the differential and integration operations. The toolbox is easy to use, as we have seen in solving the Fourier transform above. Apart from this, this toolbox also finds the Laplace transform of the various equations. Also, this toolbox has a wide application in the science and engineering field.
```

Comment:
1. First sentence. Rewrite to "Symbolic toolbox is a very important toolbox in MATLAB in solving differential and integration operations."
2. Second sentence. Rewrite to "It is easy to use as we have seen in solving the Fourier series above." I believe it's Fourier series not Transform. They are different, right?
3. Third sentence- "Apart from this, this toolbox helps find the Laplace transform of various equations."
4. Last sentence - "Generally, this toolbox has a wide application in the science and engineering field."

For the conclusion part, it is always nice to summarise what you have talked about in the article in order.

More importantly, talk about how the Symbolic Math toolbox has helped analyze the Fourier series. Or how it helps. You can do this before the conclusion or at the introduction.